utils/pow.h

```diff
@@ -0,0 +1,26 @@
+#ifndef UTILS_POW_H_
+#define UTILS_POW_H_
+
+#include <stdint.h>
+#include "common/model/bundle.h"
+#include "types/types.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+void pow_init();
+
+void pow_destroy();
+
+flex_trit_t* ta_pow_dcurl(const flex_trit_t* const trits_in, const uint8_t mwm);
```

Comment:
I dislike the name `ta_pow_dcurl` where `dcurl` should be one of runtime/build-time dependency. Therefore, `ta_pow_dcurl` can be declared as local symbol and hooked by implementation table. That is, we shall split `ta_pow` interface and implementation.

Reply:
It is not necessary to expose.